library/swift/test/EngineBuilderTests.swift

```diff
@@ -199,6 +199,20 @@ final class EngineBuilderTests: XCTestCase {
     self.waitForExpectations(timeout: 0.01)
   }
 
+  func testAddingStringAccessorToConfigurationWhenRunningEnvoy() throws {
+    let expectation = self.expectation(description: "Run called with expected data")
+    MockEnvoyEngine.onRunWithConfig = { config, _ in
+      XCTAssertEqual(1, config.stringAccessors.count)
```

Comment:
A better test might be doing `XCTAssertEqual("hello", config.stringAccessors. getEnvoyString())`